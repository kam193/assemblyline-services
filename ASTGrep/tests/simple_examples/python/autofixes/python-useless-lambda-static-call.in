x = lambda ooOOODDOODODooOOoD:vars()
y = lambda ooOOODDOODODooOOoD:vars()['xcz']
z = lambda ooOOODDOODODooOOoD:vars().aaa
z2 = lambda ooOOODDOODODooOOoD:vars(a)
