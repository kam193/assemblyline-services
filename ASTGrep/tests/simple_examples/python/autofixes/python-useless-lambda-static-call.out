x = vars
y = lambda ooOOODDOODODooOOoD:xcz
z = lambda ooOOODDOODODooOOoD:vars().aaa
z2 = lambda ooOOODDOODODooOOoD:vars(a)
